lettuce/flows/channel.py

```diff
@@ -0,0 +1,53 @@
+"""
+Cannel flows in 2D.
+"""
+
+import numpy as np
+import torch
+
+from lettuce.unit import UnitConversion
+
+
+class ChannelFlow2D(object):
+    def __init__(self, resolution, reynolds_number, lattice):
+        self.resolution = resolution
```

Comment:
Add some point this should be general enough to take the magnitude of the force. But for now fixing the values is OK.